packages/apps/immutable-records/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@kadena/immutable-records",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "build": "next build",
+    "dev": "next dev",
+    "format": "npm run format:pkg && npm run format:src",
+    "format:ci": "prettier src --check",
+    "format:pkg": "sort-package-json",
+    "format:src": "prettier src --write",
```

Comment:
Please `format:md` like the others.

Reply:
This gives me the following, I'll leave it out for now bug can investigate later.
![image](https://github.com/kadena-community/kadena.js/assets/45095973/bfb7794d-6841-49f3-9d5e-ecc8af25ed5a)

Reply:
You can add `"@kadena-dev/markdown": "workspace:*",` to devDeps. It's more correct to also add `remark-cli`, but you can omit it for now.